src/main/java/com/fasterxml/jackson/databind/ObjectMapper.java

```diff
@@ -3637,6 +3637,8 @@ public <T extends JsonNode> T valueToTree(Object fromValue)
      *  serializable)
      *
      * @deprecated Since 2.18 use discouraged; method to be removed from Jackson 3.0
+     * It is recommended to try directly testing de/serialization using actual JSON inputs and outputs
```

Comment:
Instead of "de/serialization" let's spell out either "serialization" or "deserialization" depending on which one method relates to (so "serialization" in this case).

Reply:
Yeah, let's do that! (done)